projects/media-viewer/src/lib/toolbar/main-toolbar/left-pane/left-pane.component.html

```diff
@@ -25,11 +25,15 @@
          tabindex="0" data-l10n-id="page" (change)="onPageNumberInputChange(pageNumberInput.value)" #pageNumberInput>
     <span id="numPages" class="toolbarLabel"></span>
   </ng-container>
-  <div class="toolbarButtonSpacer"></div>
-  <button *ngIf="toolbarButtons.showHighlightButton" (click)="onClickHighlightToggle()" [class.toggled]="toolbarEvents.highlightMode | async" class="toolbarButton highlightBtn" title="Highlight Text Mode" id="toggleHighlightButton" tabindex="0" data-l10n-id="toggleHighlightButton">
-    <span data-l10n-id="next_label">Highlight Text</span>
-  </button>
-  <button (click)="onClickDrawToggle()" *ngIf="toolbarButtons.showDrawButton" [class.toggled]="toolbarEvents.drawMode | async" class="toolbarButton drawBtn" title="Draw Mode" id="toggleDrawButton" tabindex="0" data-l10n-id="toggleDrawButton">
-    <span data-l10n-id="next_label">Draw</span>
-  </button>
+
+</div>
+<div id="annotationButtons">
+  <div class="splitToolbarButton hidden1000px" >
```

Comment:
this is just nitpicking but maybe change the name as 1000px isnt the best thing.

Reply:
there already are names like hiddenLargeView, hiddenMediumView, ... etc that's why I wanted to be specific

Reply:
it's not clear from the name whether it's hidden for width higher or lower than 1000px

Reply:
I will rename